packages/terra-status-view/tests/wdio/status-view-spec.js

```diff
@@ -0,0 +1,189 @@
+/* global browser, Terra, before */
+const shouldTheme = (customProperties) => {
```

Comment:
This wont' work anymore as `browser.setCSSCustomProps` has been removed. Use

```Terra.should.themeEachCustomProperty``` instead. See the [badge test](https://github.com/cerner/terra-core/blob/master/packages/terra-badge/tests/wdio/badge-spec.js#L10)s for an example